app-creator/src/widgets/tool-bar/test/tool-bar_test.ts

```diff
@@ -0,0 +1,26 @@
+import {ToolBar} from '../tool-bar';
+import {fixture, html} from '@open-wc/testing';
+
+const assert = chai.assert;
+
+suite('tool-bar', () => {
+  test('is defined', () => {
+    const el = document.createElement('tool-bar');
+    assert.instanceOf(el, ToolBar);
+  });
+
+  test('renders with default values', async () => {
+    const el = await fixture(html`<tool-bar></tool-bar>`);
+    assert.shadowDom.equal(
+      el,
+      `
+      <div id="container">
+        <h3>
+          <strong id="app-title-prefix">Google Earth Engine</strong>
```

Comment:
Declare these two as constants in the component and check that the text content of the shadowDom contains them, instead of testing the literal shadowDOM strings, as explained above.